common/Unit.cpp

```diff
@@ -388,7 +388,11 @@ int UnitBase::uninit()
         dlclose(DlHandle);
     DlHandle = nullptr;
 
+#ifndef NDEBUG
     return GlobalResult == TestResult::Ok ? EX_OK : EX_SOFTWARE;
+#else // !NDEBUG
+    return EX_OK; // Always success in release.
```

Comment:
Can we return much earlier here - the dlopen thing is potentially regression / risky - we should never dlopen if we're not in debug mode ?

Reply:
Done. Excluded `dlopen` and `dlclose` altogether.